lib/Linenoise.pm6.in

```diff
@@ -12,7 +12,19 @@ module Linenoise:ver<0.1.1>:auth<github:hoelzro> {
     my constant O_NONBLOCK   = #`(FILL-ME-IN);
     my constant CLIB         = #`(FILL-ME-IN);
 
-    my constant LIBLINENOISE = %?RESOURCES<libraries/linenoise>.Str;
+    my constant LIBLINENOISE = do if $*DISTRO.is-win {
+        my constant LIBLINENOISE = do {
+            my Str $site-path=#`(FILL-THIS);
```

Comment:
This line fails to compile, and it actually looks like an error. Maybe missing quotes?

Reply:
Hmm. It compiles on Windows without errors. What is the error?